packages/compass-import-export/src/components/import-preview.tsx

```diff
@@ -109,29 +173,108 @@ function ImportPreview({
                     onFieldCheckedChanged(field.path, !!e.target.checked)
                   }
                 />
-                <div>
-                  <Label
-                    id={`toggle-import-field-label-${field.path}`}
-                    className={fieldPathHeaderStyles}
-                    htmlFor={`toggle-import-field-checkbox-${field.path}`}
-                  >
-                    <span title={field.path}>{field.path}</span>
-                  </Label>
-                  <SelectFieldType
-                    fieldPath={field.path}
-                    selectedType={field.type}
-                    onChange={(newType: string) =>
-                      setFieldType(field.path, newType)
-                    }
+                <Label
+                  id={`toggle-import-field-label-${field.path}`}
```

Comment:
It was like this before, looks like this id doesn't match the `aria-labelledby` on the checkbox.  Can we align them in these changes?